sway-lsp/src/capabilities/inlay_hints.rs

```diff
@@ -43,7 +43,7 @@ pub fn inlay_hints(
 
     let hints: Vec<lsp_types::InlayHint> = session
         .token_map()
-        .tokens_for_file(engines.se(), uri)
+        .tokens_for_file(uri)
```

Comment:
Nice to not have to pass in the source engine everywhere :)

Reply:
sure is :)